tests/PackageSecurityExtensionTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+class PackageSecurityExtensionTest extends SapphireTest
+{
+    protected static $fixture_file = 'PackagesWithAlerts.yml';
+
+    public function setUp()
+    {
+        parent::setUp();
+        $manifest = new SS_ConfigManifest(BASE_PATH, true);
+        $moduleDoesNotExist = $manifest->moduleExists('bringyourownideas/silverstripe-maintenance');
+        if ($moduleDoesNotExist) {
+            $this->markTestSkipped(
+                'Module bringyourownideas/silverstripe-maintenance is required for this test, but is not present.'
+            );
+        }
+    }
+
+    public function testListSecurityAlertIdentifiers()
+    {
+        $package = $this->objFromFixture('Package', 'otheralerts');
+        $this->assertEquals('ABC-001, SPY-007', $package->listSecurityAlertIdentifiers());
+    }
+
+    public function testGetBadgesHook()
+    {
+        $package = $this->objFromFixture('Package', 'otheralerts');
```

Comment:
Can you please use `Package::class` to minimise migration concerns against 2.x-dev?